data/latest/test-refresh-config-paths.sh

```diff
@@ -12,42 +12,41 @@ SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
 source "$SCRIPT_DIR/utils.sh"
 
 DEFAULT_TEST_CHANNEL=${DEFAULT_TEST_CHANNEL:-beta}
+EDGEX_STABLE_CHANNEL="2.0/stable"
```

Comment:
I think it is better to call this the `EDGEX_PREV_STABLE_CHANNEL` to distinguish `EDGEX_STABLE_CHANNEL` in run-all-tests.

Reply:
I agree. Updated.